src/main/kotlin/racingcar/Car.kt

```diff
@@ -0,0 +1,51 @@
+package racingcar
+
+private const val NAME_LENGTH_MAX = 5
+private const val ACCELERATE_MIN_VALUE = 0
+private const val ACCELERATE_MAX_VALUE = 9
+private const val ACCELERATE_STANDARD_VALUE = 4
+
+class Car(_name: String, _position: Int = 0) {
+    val name: String = _name
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
class Car(val name: String, _position: Int = 0) {
```